packages/eslint-plugin-pf-codemods/src/rules/v6/noDuplicateImportSpecifiers/no-duplicate-import-specifiers.test.ts

```diff
@@ -0,0 +1,64 @@
+const ruleTester = require("../../ruletester");
+import * as rule from "./no-duplicate-import-specifiers";
+
+ruleTester.run("no-duplicate-import-specifiers", rule, {
+  valid: [
+    {
+      // we care only about imports from "@patternfly/react-core"
+      code: `import { Button, Button } from "somewhere"`,
+    },
+    {
+      code: `import { Button, Button as AnotherButton } from "@patternfly/react-core";
```

Comment:
Should we preserve imports of the same thing with different local names like this? 🤔

As an example I feel like if someone has `import { Text, TextContent as TC}` it should become `import { Content }` and the local name removed, not `import { Content, Content as TC }`.

I won't block either way though since it is valid JS.

Reply:
Ah good point!